src/routes/layout.css

```
@@ -32,45 +32,40 @@
-	--color-gold-dark: #765a24;
-	--color-sage: #7a9a8a;
-	--color-sage-dark: #4d7561;
-	--color-border: #333333;
-	--font-sans: 'Outfit', ui-sans-serif, system-ui, sans-serif;
-	--font-serif: 'Forum', ui-serif, Georgia, serif;
-}
-
 :root {
-	font-family: var(--font-sans);
+	font-family:
+		'Outfit',
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)